DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -55,6 +56,10 @@ public class DcAccounts {
         dc_accounts_maybe_network_lost(accountsPointer)
     }
 
+    public func isAllWorkDone() -> Bool {
+        return dc_accounts_all_work_done(accountsPointer) != 0
```

Comment:
btw. I wasn't aware we have this API !

Reply:
i was involved in the design that time, but i also forgot about that :)